src/apps/activity_assignments/errors.py

```diff
@@ -4,4 +4,16 @@
 
 
 class ActivityAssignmentActivityOrFlowError(ValidationError):
+    message = _("Either activity_id or activity_flow_id must be provided, but not both")
+
+
+class ActivityAssignmentNotActivityAndFlowError(ValidationError):
     message = _("Either activity_id or activity_flow_id must be provided")
+
+
+class ActivityAssignmentMissingRespondentError(ValidationError):
+    message = _("Respondent subject ID must be provided")
+
+
+class ActivityAssignmentMissingTargetError(ValidationError):
+    message = _("Target subject ID must be provided")
```

Comment:
I'm guessing these extra error classes are probably not needed, pending other discussions here.

Reply:
yeah, we don't need them